tests/wandb_integration_test.py

```diff
@@ -424,49 +424,43 @@ def test_end_to_end_preempting_via_module_func(live_mock_server):
 
 @pytest.mark.flaky
 @pytest.mark.xfail(platform.system() == "Windows", reason="flaky test")
-def test_live_policy_file_upload(live_mock_server, test_settings, mocker):
+def test_live_policy_file_upload(live_mock_server, test_settings):
     test_settings.update(
-        {"start_method": "thread"},
+        {
+            "start_method": "thread",
```

Comment:
i see, that is how this test was working in the past.. it was relying on thread mode.